unity_jobs_simulator/Assets/Scripts/Robot_Controller.cs

```diff
@@ -154,27 +156,27 @@ public void UpdateCurrentDriveTargetAndJointAngles()
         }
     }
 
-    public void GetIncrements()
+    public void GetAndSetIncrements()
     {
         _increments.Clear();
         for(int i=0 ; i < TargetJointAngles.Count ; i++)
         {
             float diff = Math.Abs(TargetJointAngles[i] - _currentDriveTarget[i]); // We get the magnitude of the angle needed to move
             float t = diff/_maxVelRad; // The time taken is the angle divided by the angular velocity
```

Comment:
diff is in degrees ( TargetJointAngles and _currentDriveTarget are both in degrees) and velocity is in radians. These should be represented in the same format, for accurate calculations.